### PR TITLE
Updated architecture with new areas

### PR DESCRIPTION
**Pull request by sirinoks**
### Documentation update
Basically, replaced an image with an updated structure. 

## Changes:
Old:

New: 

- Added Queue (the best I understand it). It was not yet said what technology will be used in it, it will need to be updated once we have some code. 
- Added names of areas in format of (Area). Areas are outlined in [this doc](https://github.com/Seneca-CDOT/starchart/wiki/Initial-Task-List-and-Scope#tasks)
Basically, trying to keep our naming of things consistent.

---

If you disagree with where things are placed, please let me know. I'm not very sure on Queue myself.

### CODE REVIEW

**Review by dadolhay — APPROVED** [review]
Looks good!

Note:
The queue will probably be this:
https://docs.bullmq.io/

**Comment by Genne23v** [comment]
Awesome diagram. I think it's good to update this PR as decision on queue will be made soon.

**Review by cychu42 — DISMISSED** [review]
This looks great!

**Comment by sirinoks** [comment]
Updated queue with technologies mentioned. bullMQ is on Redis, so might as well add that for better understanding.

**Review by Eakam1007 — APPROVED** [review]
Looks good!

**Comment by sirinoks** [comment]
I'm currently also working on #141. So I'm making labels and this diagram consistent with also [this doc](https://github.com/DevelopingSpace/starchart/wiki/Initial-Task-List-and-Scope). Keep in mind, the label colors are the same as in this image.

**Comment by sirinoks** [comment]
Too many changes while I'm doing this, will temporarily make it a draft.

**Review by Genne23v — APPROVED** [review]
LGTM!

**Review by cychu42 — APPROVED** [review]
